src/sys/socket/mod.rs

```diff
@@ -414,6 +414,14 @@ impl<'a> Iterator for CmsgIterator<'a> {
                 Some(ControlMessage::ScmTimestamp(
                     &*(cmsg_data.as_ptr() as *const _)))
             },
+            #[cfg(any(target_os = "linux", target_os = "android", target_os = "macos", target_os = "ios"))]
```

Comment:
These lists should be alphabetized (android, ios, linux, macos). This applies to all of your `#[cfg` attributes.